packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -84,6 +84,35 @@ class ExpansionPanel {
   ///
   /// Defaults to false.
   final bool isExpanded;
+
+}
+
+/// An expansion panel that allows for radio-like functionality.
+///
+/// This type of panel will automatically handle closing and opening itself,
+/// set the [initializesExpanded] value to true on a panel if you want it to begin
```

Comment:
I'm not sure about placing `initializesExpanded` on the RadioPanel. Try adding another named argument to the ExpansionPanelList.radio constructor which takes a panel or value of a panel to open "only the first time it is created". Then don't both updating in the future, it doesn't make sense to try and keep all of the initializesExpanded and state changed by user interaction in sync.